src/components/analytics/AnalyticsCharts.jsx

```diff
@@ -12,32 +11,53 @@ import AnalyticsApiService from './data/service';
 // eslint-disable-next-line no-unused-vars
 export default function AnalyticsCharts(enterpriseId) {
   const [token, setToken] = useState(null);
+  const [tokenUsedOnce, setTokenUsedOnce] = useState(false);
   const [isLoading, setIsLoading] = useState(false);
   const [error, setError] = useState(null);
-  // Fetch token
+  const ref = useRef(null);
+  let tableauUrl = 'https://enterprise-tableau.edx.org/views/enterpriseadminanalytics/enroll_dash';
```

Comment:
shouldn't we use `configuration.TABLEAU_URL` rather than hardcoding the base url?

Reply:
The one from the remote config will overwrite this value.